tests/rules/no-empty-collection.test.ts

```diff
@@ -247,7 +253,32 @@ ruleTester.run('Empty collections should not be accessed or iterated', rule, {
               console.log(array[2]);
               console.log(array[2]);
             }`,
-      errors: 4,
```

Comment:
it's not possible anymore to define a number as the number of errors. You can only specify the entire error itself.